koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -246,20 +246,22 @@ def should_s3_parquet_be_cleared(self, manifest: CostUsageReportManifest) -> boo
         )
         return result
 
-    def get_s3_parquet_cleared(self, manifest: CostUsageReportManifest, report_type: str = None) -> bool:
+    def get_s3_parquet_cleared(self, manifest: CostUsageReportManifest, file_stem: str = None) -> bool:
         """Return whether we have cleared CSV files from S3 for this manifest."""
         if not manifest:
             return False
-        if manifest.cluster_id and report_type:
-            return manifest.s3_parquet_cleared_tracker.get(report_type)
+        if manifest.cluster_id and file_stem:
+            filename_nocounter = file_stem.rsplit(".", 1)[0]
+            return manifest.s3_parquet_cleared_tracker.get(filename_nocounter)
         return manifest.s3_parquet_cleared
 
-    def mark_s3_parquet_cleared(self, manifest: CostUsageReportManifest, report_type: str = None) -> None:
+    def mark_s3_parquet_cleared(self, manifest: CostUsageReportManifest, file_stem: str = None) -> None:
         """Mark Parquet files have been cleared from S3 for this manifest."""
         if not manifest:
             return
-        if manifest.cluster_id and report_type:
-            manifest.s3_parquet_cleared_tracker[report_type] = True
+        if manifest.cluster_id and file_stem:
+            filename_nocounter = file_stem.rsplit(".", 1)[0]
+            manifest.s3_parquet_cleared_tracker[filename_nocounter] = True
```

Comment:
This method shouldn't be responsible for ensuring the filename is correct. Leaving that up to the caller makes this method more flexible and less surprising.
```suggestion
            manifest.s3_parquet_cleared_tracker[file_stem] = True
```

Reply:
move it here:
https://github.com/project-koku/koku/pull/4685/files#diff-f9c85bc4f2f7812a9afad872e5a4a3a49ff37e64b0aeeec5d1422fe3fae9d827R329-R331